lib/rules/no-invalid-double-slash-comments/index.mjs

```diff
@@ -28,22 +28,36 @@ const rule = (primary) => {
 					node: decl,
 					result,
 					ruleName,
-					word: decl.toString(),
 				});
 			}
 		});
 
 		root.walkRules((ruleNode) => {
-			for (const selector of ruleNode.selectors) {
+			// ruleNode.selectors elements are trimmed
+			const selectors = ruleNode.selector.split(',');
+			let index = 0;
+
+			for (const value of selectors) {
+				const selector = value.trimStart();
+
 				if (selector.startsWith('//')) {
+					const offset = value.length - selector.length;
+					const i = index + offset;
+					const lines = ruleNode.toString().slice(i).split('\n');
+					const comment = lines[0];
+					const word = lines.length > 1 ? `${comment}\n` : comment;
+
 					report({
 						message: messages.rejected,
 						node: ruleNode,
 						result,
 						ruleName,
-						word: ruleNode.toString(),
+						index: i,
+						word,
```

Comment:
[ask] Does it make sense to specify `endIndex` instead of `word` because `index` is already set?

Reply:
It's more complicated than using `word`.
This will eventually be refactored again in the upcoming weeks so it's acceptable for now.

Reply:
My bad I was wrong. It will be addressed in an upcoming PR.